app/scripts/background.js

```diff
@@ -404,23 +405,11 @@ async function loadPhishingWarningPage() {
  */
 export async function loadStateFromPersistence() {
   // migrations
-  const migrator = new Migrator({ migrations });
   migrator.on('error', console.warn);
 
   // read from disk
   // first from preferred, async API:
-  versionedData =
-    (await localStore.get()) || migrator.generateInitialState(firstTimeState);
-
-  // check if somehow state is empty
-  // this should never happen but new error reporting suggests that it has
-  // for a small number of users
-  // https://github.com/metamask/metamask-extension/issues/3919
-  if (versionedData && !versionedData.data) {
```

Comment:
Is this check for `!versionedData.data`, or an alternative to it, present in the new `ExtensionStore.ts`? I can't find it.

Reply:
Addressed by 70e61917c9